contrib/windows/build-installer.nsi

```diff
@@ -29,7 +29,7 @@ FunctionEnd
 Function createDesktopLink
   ${NSD_GetState} $Checkbox $0
   ${If} $0 <> 0
-    CreateShortCut "$DESKTOP\julia.lnk" "$INSTDIR\bin\julia.exe"
+    CreateShortcut "$DESKTOP\julia.lnk" "$INSTDIR\mintty\usr\bin\env.exe" "HOME=%HOMEDRIVE%%HOMEPATH% $INSTDIR\mintty\usr\bin\mintty.exe -h error -i $INSTDIR\bin\julia.exe $INSTDIR\bin\julia.exe" "$INSTDIR\bin\julia.exe" 0 SW_SHOWNORMAL "" "The Julia REPL"
```

Comment:
I like having `-h error`. Can you test how long the path has to get before it's actually an issue?